azkaban-hadoop-security-plugin/src/main/java/azkaban/security/HadoopSecurityManager_H_2_0.java

```diff
@@ -780,4 +789,27 @@ private Text getMRTokenRenewerInternal(final JobConf jobConf) throws IOException
         hsProxy.getConnectAddress());
   }
 
+  // we use this function to create retrying metastore client
```

Comment:
Use Javadoc format for method level comments; it would be better to say, "Method to create a metastore client that retries on failures". Update the method name to be createRetryingMetastoreClient